django/website/logframe/tests/test_period_utils.py

```diff
@@ -0,0 +1,55 @@
+from datetime import date, timedelta
+
+from ..period_utils import get_month_shift, get_periods, periods_intersect
+
+
+def test_get_month_shift_handles_december():
```

Comment:
If we are going to test the december edge case, maybe also have a general test to demonstrates how get_month_shift works.